doc/recipe/preprocessor.rst

```diff
@@ -647,8 +647,10 @@ to observational data, these biases have a significantly lower statistical
 impact when using a multi-model ensemble. ESMValTool has the capability of
 computing a number of multi-model statistical measures: using the preprocessor
 module ``multi_model_statistics`` will enable the user to ask for either a
-multi-model ``mean``, ``median``, ``max``, ``min`` and / or ``std`` with a set
-of argument parameters passed to ``multi_model_statistics``.
+multi-model ``mean``, ``median``, ``max``, ``min``, ``std``, and / or ``pXX``
+with a set of argument parameters passed to ``multi_model_statistics``.
+Percentiles can be specified like ``p10`` or ``p90``. ``p995`` will be
+interpreted as the 99.5-th percentile.
```

Comment:
It would be better to include the period in the percentile, what if I want the 1.5th percentile? It will get interpreted as 15th with this notation. In the resulting filename you could then replace the '.' by a '-'.

Reply:
I like the idea, so I tried to change `statistic_product.filename`, but it appears it is not possible to overwrite the filename of a `PreprocessorFile`. So how should I go about changing this?

Reply:
Change this line
https://github.com/ESMValGroup/ESMValCore/blob/c5ffda0c66292a0a7d594b2e4b6e3478cde00075/esmvalcore/_recipe.py#L632-L633
to
```python
        attributes['dataset'] = attributes['alias'] = 'MultiModel{}'.format(
            statistic.title().replace('.', '_')
```